BUILD.gn

```diff
@@ -162,17 +162,35 @@ if (!is_mac) {
 
     sources = []
     foreach(locale, locales) {
-      # print (locale)
-      if (is_mac) {
```

Comment:
this is just cleanup.
this `if (is_mac)` doesn't make sense because this is already in `if (!is_mac)`.